app/presenters/idv/in_person/ready_to_verify_presenter.rb

```diff
@@ -31,7 +31,7 @@ def formatted_due_date
       def selected_location_hours(prefix)
         return unless selected_location_details
         hours = selected_location_details["#{prefix}_hours"]
-        return localized_hours(hours) if hours
+        return UspsInPersonProofing::EnrollmentHelper.localized_hours(hours) if hours
```

Comment:
can remove the redundant `return` while we're here

```suggestion
        UspsInPersonProofing::EnrollmentHelper.localized_hours(hours) if hours
```